pkg/status/templates/logsagent.tmpl

```diff
@@ -61,7 +61,10 @@ Logs Agent
         {{ $message }}
       {{- end }}
       {{- if .inputs }}
-      Inputs: {{ range $input := .inputs }}{{$input}} {{ end }}
+      Inputs:
+      {{- range $input := .inputs }}
```

Comment:
Should also make this change in `generalStatus.tmpl` (used in the web UI). 

Probably out of scope for this change given how simple this is but in the future we should replace `Inputs` and some of the other key/value information here with `InfoProvider` interface I added for `Source` which will handle the formatting automatically and not require changes to the tmpl files anymore. (see how `.info` is unpacked below)